File: 6KYU/Simple Fun #52: Pair of Shoes/solution.py
def pair_of_shoes(shoes):
   leftRight = 0
   Size = 0
    
   for shoe in shoes:
       if shoe[0] == 0:
           leftRight = leftRight + 1
           Size = Size + shoe[1]
       else:
           leftRight = leftRight - 1
           Size = Size - shoe[1]
   
   return leftRight == 0 and Size == 0

File: 6KYU/Simple Fun #52: Pair of Shoes/solution.cs
namespace myjinxin
{
    using System;
    
    public class Kata
    {
        public bool PairOfShoes(int[][] shoes){
          int leftRight = 0;
          int sizes = 0;
          
          foreach(var shoe in shoes)
          {
            if(shoe[0] == 0)
            {
              leftRight++;
              sizes+=shoe[1];
            }
            if(shoe[0] == 1)
            {
              leftRight--;
              sizes-=shoe[1];
            }
          }
          return leftRight == 0 && sizes == 0;
        }
    }
}